nautobot/core/templates/inc/nav_menu.html

```diff
@@ -13,795 +13,52 @@
+                            <li class="dropdown">
+                                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                    {{ tab_name }} <span class="caret"></span>
+                                </a>
+                                <ul class="dropdown-menu">
+                                    {% for group_name, group_details in tab_details.groups.items %}
+                                        {% if request.user|has_perms:group_details.permissions or not settings.HIDE_RESTRICTED_UI %}
+                                            <li class="dropdown-header">{{ group_name }}</li>
+                                            {% for item_link, item_details in group_details.items.items %}
+                                                {% if request.user|has_perms:item_details.permissions or not settings.HIDE_RESTRICTED_UI %}
+                                                    <li {% if not request.user|has_perms:item_details.permissions %} class="disabled"{% endif %}>
+                                                        <div class="buttons pull-right">
+                                                            {% for button in item_details.buttons %}
+                                                                {% if not button.permissions or request.user|has_perms:button.permissions %}
+                                                                    <a href="{% url button.link %}" class="btn btn-xs btn-{{ button.button_class }}" title="{{ button.title }}"><i class="{{ button.icon_class }}"></i></a>
+                                                                {% endif %}
+                                                            {% endfor %}
+                                                        </div>
+                                                        <a href="{% url item_link %}">{{ item_details.link_text }}</a>
+                                                    </li>
+                                                {% endif %}
+                                            {% endfor %}
+                                            {% if not forloop.last %}
+                                                <li class="divider"></li>
+                                            {% endif %}
+                                        {% endif %}
+                                    {% endfor %}
+                                </ul>
+                            </li>
+                        {% endif %}
+                    {% endfor %}
+                </ul>
             {% endif %}
 
+            <ul class="nav navbar-nav">
                 {% if registry.plugin_menu_items %}
                     {% include 'inc/plugin_menu_items.html' %}
                 {% endif %}
             </ul>
-            {% endif %}
```

Comment:
Did this move the plugin menus out from within the `if request.user.is_authenticated` block?